### DISCUSSION TITLE
How to link to an header in a content tab?

### DISCUSSION THREAD

**Discussion by hpz24** [requarks/wiki#6846] [Help / Questions] [answered]
Hi, 
I´m using the content tab with 3 tabs and I want to link form another page (or another place on the same page) to exactly one of those different headers in the tab.
So if we stay on the example from the docs (lets call the page "content"):

`
# Tabs {.tabset}
## First Tab
Any content here will go into the first tab...
## Second Tab
Any content here will go into the second tab...
## Third Tab
Any content here will go into the third tab...
`

So for my understanding, when I try with 
`[superfancykeyword](./content#second-tab)`

then it does not work (means that nothing happens wenn I click on that link). The linking to header in that form works of course for "normal" headers but not for that one in the content tab.

Am I doing something wrong?
Thanks and
Regards

**Comment by NGPixel** [comment]
You can't link to a tab itself, as they are not headers in the rendered version. You need to add a header inside the tab, e.g.:

```
# Tabs {.tabset}
## First Tab
abc

## Second Tab
### Something
abc
### Nothing
def

## Third Tab
abc
```

You can then link to `#something` and `#nothing`.

**Comment by hpz24** [comment]
Hi, 
thanks for your reply!
I tried it out and it works - as long as the linked tab is active (when linking from the same page).
If the target tab is not active and another tab is visible at the moment, the link doesn´t work.

**Reply by NGPixel** [reply]
Are you running the latest version? It definitely works on non-active tabs too

**Comment by hpz24** [comment]
I´ve running 2.5.300

I tried it now with a new fresh wiki-site and can confirm that it works only if the active tab is visible.

```
<filename: contentlink>

## Tabs {.tabset}
### First Tab
#### link1
Any content here will go into the first tab...

### Second Tab
#### link2
Any content here will go into the second tab...

### Third Tab
Any content here will go into the second tab...

## Text
Then a lot of lorem ipsum text as placeholder...
``` 

At the end my two testlinks:

```
[link1](./contentlink#link1)
[link2](./contentlink#link2)
``` 

Depending on whether tab1 or tab2 is visible, the browser only ever jumps to the already visible heading in the active tab.

**Reply by NGPixel** [reply, marked as answer]
From the same page sure, but linking from another page should switch to the proper tab on load.

**Reply by hpz24** [reply]
Okay, there was the problem :)
Thx
